apps/mochi-web/components/Profile/PaylinkSection.tsx

```diff
@@ -43,7 +43,9 @@ export const PaylinkSection = () => {
           {
             header: 'Wen',
             accessorKey: 'created_at',
-            accessorFn: (row) => format(new Date(row.created_at ?? ''), 'Pp'),
+            accessorFn: (row) =>
+              formatDate(row.created_at ?? '', 'dd/MM/yyyy hh:mma'),
```

Comment:
Maybe it's safer to hide the data instead of showing the wrong info

```suggestion
            row.created_at ? formatDate(row.created_at, 'dd/MM/yyyy hh:mma') : null,
```